src/CommunityVoices/App/Api/View/DisplayError.php

```diff
@@ -23,4 +23,14 @@ protected function getError($request)
 
         return $response;
     }
+    protected function getAllErrors($request)
+    {
+        $errors = explode('[]',str_replace('\n','',file_get_contents('/var/www/html/log/access.log')));
+        foreach($errors as &$item) {
+            $item = array("item" => $item);
+        }
+        $response = new HttpFoundation\JsonResponse(["errors" => $errors]);
```

Comment:
From the perspective of consistency, I think it would be better to name the key something else (maybe `errorLog`?).  The reasoning is that `errors` is what we use for true errors that occurred on the API side, whereas this array is a group of desired output by the program.